guides/deploying.md

```diff
@@ -0,0 +1,61 @@
+# Deploying
+
+This is a guide to how we deploy changes to code onto different environments.
+
+## WordPress projects on GovPress
+
+This guide goes through the process of deploying to the different environments on GovPress. On GovPress there are two main environments: `testing` and `production`.
+
+This guide assumes that the main working repo is on Github and the repo for deployments is on GovPress. Your local repo should have 2 remotes set up to work with:
+- `origin` - the main Github repo
+- `govpress` - the GovPress deployment repo
+
+### Deploying to Testing
+
+You can deploy any branch to testing without going through a code review process. Testing is managed from the `stage`  branch on the `origin` repo, but can be overridden at any time for specific testing requirements. 
+
+Deploying from `staging` using [Git ENV](https://github.com/dxw/playbook/blob/master/guides/git-env.md):
```

Comment:
The relative links work through the repo method, but when compiling to HTML we need the full links, as the guides aren't being compiled as far as I know.